.github/workflows/scaladoc.yaml

```diff
@@ -64,9 +64,9 @@ jobs:
         with:
           inlineScript: |
             DOC_DEST=pr-${PR_NUMBER:-${GITHUB_REF##*/}}
-            echo uplading docs to https://scala3doc.virtuslab.com/$DOC_DEST
-            az storage container create --name $DOC_DEST --account-name scala3docstorage --public-access container
-            az storage blob sync -s scala3doc/output -c $DOC_DEST --account-name scala3docstorage
+            echo uplading docs to https://scaladoc.virtuslab.com/$DOC_DEST
```

Comment:
Is this intentional? The domain doesn't seem available yet...

Reply:
I've used replace to change scala3doc to scaladoc and in some places (like this one) rename should not be applied